src/js/components/Button/index.d.ts

```diff
@@ -14,6 +14,7 @@ export interface ButtonProps {
   gap?: "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | string;
   hoverIndicator?: boolean | string | "background" | {background?: boolean | string};
   href?: string;
+  target?: "_self" | "_blank" | "_parent" | "_top";
```

Comment:
We need another value "string" here to support all prop types.

Reply:
Or wait, actually https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a says target can have only 4 values so looks like string goes away from prop types. What do you think?

Reply:
Both approaches (loose vs strict 4 values) work for me. And yea that's the docs that I also checked before PRing this :)
Will try to repush to see whether the pipelines get fixed, but other than that @crashuniverse plz give me a not whether you still want me to add a `| string` or not in there.

Reply:
It seems that all tests are passing now :)

Reply:
This looks similar than `align` property in Box.js ts definitions, so I'd suggest to remove `string` from other places in the PR. Like `align` property we have covered all 4 possible values and there is no other possible string value.

Reply:
I think changes to other components are orthogonal to this one and shouldn't be combined in this PR. I'm fine with restricting the allowed values to the four specified.